app/logic/Calendar/ActiveSync/ActiveSyncIncomingInvitation.ts

```diff
@@ -39,6 +39,7 @@ export class ActiveSyncIncomingInvitation {
     this.event.myParticipation = response;
     await this.event.respondToInvitation(response, this.calendar.account); // needs 16.x to do this automatically
     await this.message.deleteMessageLocally(); // Exchange deletes the message from the inbox
+    await this.calendar.listEvents();
```

Comment:
1. Shall we be more specific and use the new `fetchFromServer()` function? Or do we not know the evffefnt yet?
2. How does the UI get updated? If the new event comes in, how does the Invitation here know to find the event and update the UI?

Reply:
1. We're going to have to Sync the calendar sooner or later, so we might as well do it now.
2. While you're still viewing the email (which should never happen, because Exchange deletes it) the UI keeps track of the response. If you get an update for that event, and view that email, the event in the calendar will be found due to its matching calUID. (This applies to all protocols.)